lively.morphic/text/morph.js

```diff
@@ -1579,6 +1579,7 @@ export class Text extends Morph {
   }
 
   get commands () {
+    if (this.readOnly) return [];
```

Comment:
We do have text morphs that are `readOnly`, yet still have a document + text layout present and make use of it. For instance in the *upon coming* component reconciliation, I sometimes set the browser source code editor to `readOnly = true` but still select/highlight ranges of code.
So I believe directly checking for the document is the better approach here.

Reply:
Do you have an example for a command which you would like to execute in this state? I.e., where the browser source editor is set to be readonly?
In the case that one cannot edit text, de facto none of the commands make sense.